### ISSUE TITLE
Storing files in different folders

### ISSUE THREAD

**Issue by kolyamba2105** [nestjs/nest#1243]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[*] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Hi,
Im am using multer package for uploading blobs on my server and I noticed that MulterModule is initialized only once while app start. However I import MulterModule in more than one of my feature modules and I expect MulterModule to behave differently in each feature module.
<br>
<img width="702" alt="2018-10-27 22 48 14" src="https://user-images.githubusercontent.com/33033411/47609192-7030ba00-da3a-11e8-96b8-fa1b86eedad8.png">
<br>
<img width="751" alt="2018-10-27 22 51 02" src="https://user-images.githubusercontent.com/33033411/47609209-cd2c7000-da3a-11e8-9f45-a78324aa5c86.png">

## Expected behavior

As it's seen on screenshots, I want files for ex. from users routes to be uploaded to "uploads/users" folder and have prefix "user - " in filename. And the same for alternatives routes: "uploads/alternatives" dest directory and "alt -" prefix for filename.
As MulterModule is initialized only once per app launch, the configuration for all routes that upload files is the same everywhere.
<br>
So, the question is: <h3>Is that possible to initialize MulterModule few times in different feature modules, so I'll have different configs?</h3>

## Environment
[System Information]
OS Version     : macOS
NodeJS Version : v8.12.0
NPM Version    : 6.4.1
[Nest Information]
common version   : 5.1.0
core version     : 5.1.0

**Comment by barretojs** [comment]
you don't need to import MulterModule!
when you apply the interceptor to the route, pass the configuration to the FileInterceptor. only register the MulterModule if you have a especific config that needs to be global to all your interceeptors and it's not in the default behavior from Nest. from the docs:

![screenshot from 2018-10-27 18-30-01](https://user-images.githubusercontent.com/43803300/47609512-701ec300-da16-11e8-9c3c-0f0f525671a3.png)

my code the shows the behavior:

![screenshot from 2018-10-27 18-27-29](https://user-images.githubusercontent.com/43803300/47609488-030b2d80-da16-11e8-9a7f-1eea7054addc.png)
![screenshot from 2018-10-27 18-27-39](https://user-images.githubusercontent.com/43803300/47609489-03a3c400-da16-11e8-9b72-9cadc52eb9dc.png)

you can do something like in the images. the result was a file "elon.png" saved on the root directory and a "elon2.png" saved on a folder called "test2" in the root directory.

**Comment by kolyamba2105** [comment]
Yes, that works! Thank you for your help and for instant response!

**Comment by barretojs** [comment]
np man! remeber to close the issue.
